src/main/java/org/dita/dost/module/BranchFilterModule.java

```diff
@@ -596,31 +596,39 @@ private void processAttributes(final Element elem, final Branch filter) {
             final String scope = elem.getAttribute(ATTRIBUTE_NAME_SCOPE);
             if ((!href.isEmpty() || !copyTo.isEmpty()) && !scope.equals(ATTR_SCOPE_VALUE_EXTERNAL)) {
                 final FileInfo hrefFileInfo = job.getFileInfo(currentFile.resolve(href));
-                final FileInfo copyToFileInfo = !copyTo.isEmpty() ? job.getFileInfo(currentFile.resolve(copyTo)) : null;
-                final URI dstSource;
-                dstSource = generateCopyTo((copyToFileInfo != null ? copyToFileInfo : hrefFileInfo).result, filter);
-                final URI dstTemp = tempFileNameScheme.generateTempFileName(dstSource);
-                final String dstPathFromMap = !copyTo.isEmpty() ? FilenameUtils.getPath(copyTo) : FilenameUtils.getPath(href);
-                final FileInfo.Builder dstBuilder = new FileInfo.Builder(hrefFileInfo)
-                        .result(dstSource)
-                        .uri(dstTemp);
-                if (dstBuilder.build().format == null) {
-                    dstBuilder.format(ATTR_FORMAT_VALUE_DITA);
+                boolean isDITA = false;
+                if(hrefFileInfo != null) {
+                    isDITA = hrefFileInfo.format == null 
+                            || ATTR_FORMAT_VALUE_DITA.equals(hrefFileInfo.format) 
+                            || ATTR_FORMAT_VALUE_DITAMAP.equals(hrefFileInfo.format);
```

Comment:
This should be extracted to method `isDitaFormat(String)`, like https://github.com/dita-ot/dita-ot/blob/6af23fe52d833c19c061cad9e748461cb59ca4bc/src/main/java/org/dita/dost/module/BranchFilterModule.java#L268